plugin-server/src/config/config.ts

```diff
@@ -131,6 +131,20 @@ export function getDefaultConfig(): PluginsServerConfig {
             ? 1024 // NOTE: ~1MB in dev or test, so that even with gzipped content we still flush pretty frequently
             : 1024 * 50, // ~50MB after compression in prod
         SESSION_RECORDING_REMOTE_FOLDER: 'session_recordings',
+        SESSION_RECORDING_REDIS_OFFSET_STORAGE_KEY: '@posthog/replay/partition-high-water-marks',
+        POSTHOG_SESSION_RECORDING_REDIS_HOST: undefined,
+        POSTHOG_SESSION_RECORDING_REDIS_PORT: 6379,
+    }
+}
+
+export const sessionRecordingBlobConsumerConfig = (config: PluginsServerConfig): PluginsServerConfig => {
+    // When running the blob consumer we override a bunch of settings to use the session recording ones if available
+    return {
+        ...config,
+        KAFKA_HOSTS: config.SESSION_RECORDING_KAFKA_HOSTS || config.KAFKA_HOSTS,
+        KAFKA_SECURITY_PROTOCOL: config.SESSION_RECORDING_KAFKA_SECURITY_PROTOCOL || config.KAFKA_SECURITY_PROTOCOL,
+        POSTHOG_REDIS_HOST: config.POSTHOG_SESSION_RECORDING_REDIS_HOST || config.POSTHOG_REDIS_HOST,
+        POSTHOG_REDIS_PORT: config.POSTHOG_SESSION_RECORDING_REDIS_PORT || config.POSTHOG_REDIS_PORT,
```

Comment:
nit, you can use `KAFKA_HOSTS: config.SESSION_RECORDING_KAFKA_HOSTS ?? config.KAFKA_HOSTS` here to translate your intent better: use the fallback if the value is `undefined`

Reply:
I see your point but I think in this case I'd rather stick with the or condition as it covers cases (that I've definitely seen) where the value is empty string as there is no way of saying it should be "null" from an environment variable.